tests/psbt.rs

```diff
@@ -0,0 +1,459 @@
+
+use std::str::FromStr;
+use std::collections::BTreeMap;
+
+use bitcoin::{Amount, Denomination, Network, OutPoint, PublicKey, PrivateKey, Script, Transaction, Txid, TxOut, TxIn, Witness};
+use bitcoin::consensus::encode::{serialize_hex, deserialize};
+use bitcoin::blockdata::{opcodes, script};
+use opcodes::all::OP_PUSHBYTES_0 as OP_0;
+use bitcoin::hashes::hex::{FromHex};
```

Comment:
We don't put the braces for single imports, one day (TM) we will have tooling to remove stuff like this :)